tools/serverpod_cli/bin/util/version.dart

```diff
@@ -0,0 +1,5 @@
+import '../generated/version.dart';
+import 'print.dart';
+
+/// Get the current version of the serverpod.
+void getVersion() => printww('Serverpod version: $templateVersion');
```

Comment:
The naming `getVersion` implies that the method returns a value. A better name would be `printVersion`. Also, please use the `() {}` method syntax unless the method is returning a value.

Reply:
Done.